cmd/app/serve.go

```diff
@@ -313,3 +349,78 @@ func checkServeCmdConfigFile() error {
 	}
 	return nil
 }
+
+func StartDuplexServer(ctx context.Context, cfg *config.FulcioConfig, ctClient *ctclient.LogClient, baseca ca.CertificateAuthority, port, metricsPort int) error {
```

Comment:
Is `host` configurable?

Reply:
viper.GetString("port")

Reply:
it was added in arg list in https://github.com/sigstore/fulcio/pull/931/files#diff-1627906dbce532aff8cc81f54790572e5ba04f3e568c88b6ed39a590642dafa4R345

Reply:
Oh whoops that was a comment I wrote awhile ago. This looks good